app/src/main/java/org/alexcode/snake/MainActivity.java

```diff
@@ -25,7 +25,7 @@ protected void onCreate(Bundle savedInstanceState) {
         playerPreferences =  new PlayerPreferences(this);
         initGameMenu();
         initPlayerData();
-        startGame.setOnClickListener(new View.OnClickListener() {
+        startGame.setOnClickListener(new View.OnClickListener() {   
```

Comment:
I think you should get rid of those spaces

Reply:
ok